quora-api/src/main/java/com/upgrad/quora/api/exception/RestExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.upgrad.quora.api.exception;
+
+import com.upgrad.quora.api.model.ErrorResponse;
+import com.upgrad.quora.api.model.SignupUserResponse;
+import com.upgrad.quora.service.exception.SignUpRestrictedException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+@ControllerAdvice
+public class RestExceptionHandler {
+
```

Comment:
@pradeepyahu can we have a comment what this handler does? You may add it at the end of project